.github/workflows/ports_cc3200.yml

```diff
@@ -0,0 +1,23 @@
+name: cc3200 port
+
+on:
+  push:
+  pull_request:
+    paths:
+      - '.github/workflows/*.yml'
+      - 'tools/**'
+      - 'py/**'
+      - 'extmod/**'
+      - 'lib/**'
+      - 'drivers/**'
+      - 'ports/cc3200/**'
+
+jobs:
+  cc3200_build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install packages
+      run: sudo apt-get install gcc-arm-none-eabi libnewlib-arm-none-eabi
```

Comment:
This is the only one which doesn't have it's setup step in ci.sh?

Reply:
Right, good catch, now fixed.  I also factored this generic gcc-arm-none-eabi set up into a reusable function.